### PR TITLE
Initial version of a simple port scanner

### PR DESCRIPTION
**Pull request by ceelias**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,10 +3,6 @@
 debug/
 target/
 
-# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
-# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
-Cargo.lock
```
Comment (ceelias) [inline .gitignore:8]: We are building an executable so we want to include the `Cargo.lock`

File: Cargo.toml [hunk 2]
```diff
@@ -6,3 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+tokio = { version = "1", features = ["full"] }
+anyhow = "1.0"
+futures = "0.3"
+clap = "2.33"
```
Comment (ceelias) [inline Cargo.toml:12]: Some of these libs are outdated and we should eval whether or not we want to use these

Comment (DeusPater) [reply Cargo.toml:12]: Okay I will check on this soon

File: .gitignore (hunk 1 above)
Comment (DeusPater) [reply .gitignore:8]: Gotcha

**Review by DeusPater — APPROVED** [review]

**Pull request merged**